checkov/terraform/checks/resource/aws/RedshiftClusterWithCommonUsernameAndPublicAccess.py

```diff
@@ -0,0 +1,25 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class RedshiftClusterWithCommonUsernameAndPublicAccess(BaseResourceCheck):
```

Comment:
Looks good! Please add an inspected key for master_username and publicly_accessible